src/App.jsx

```diff
@@ -40,7 +40,15 @@ function App() {
         rating={4.85}
         yearsHosting={1}
       />
+       <ProductCard 
+        image="https://a0.muscache.com/im/pictures/hosting/Hosting-U3RheVN1cHBseUxpc3Rpbmc6MTEzMTA4OTQ5ODA0MDcwMTE4Mw%3D%3D/original/4f7a276e-9995-4b32-bda6-300de0619b25.png?im_w=2560&im_q=highq" 
```

Comment:
we shouldn't use the images from other websites.. they might change, we should add images ourselves to our repo and link to them

Reply:
Image linked to asset in repo